contributors/devel/on-call-federation-build-cop.md

```diff
@@ -0,0 +1,159 @@
+federation control plane. These federation control planes deployed in
+separate namespaces are independent of each other and do not interfere
+with other federation control planes in anyway.
+
+### Configuration
+
+The two jobs are configured differently.
+
+#### Deploy job
+
+The deploy job is configured as a CI/Soak job in Jenkins.
+Configuration steps are described in https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/jenkins/README.md#how-to-work-with-jenkins-jobs
+
+The configuration of the deploy job is stored in:
+
+* Jenkins config: https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/jenkins/job-configs/kubernetes-jenkins/bootstrap-ci-soak.yaml#L76
+* Test job/bootstrap config: https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/jobs/config.json#L3996
+* Test grid config: https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/testgrid/config/config.yaml#L152
+* Job specific config: https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/jobs/ci-kubernetes-pull-gce-federation-deploy.env
+
+#### Test job
+
+The test job is configured in prow, but it runs in Jenkins mode. The
+configuration steps are described in https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/README.md#create-a-new-job
+
+The configuration of the test job is stored in:
+
+* Prow config: https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/prow/config.yaml#L244
+* Test job/bootstrap config: https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/jobs/config.json#L4691
+* Job specific config: https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/jobs/pull-kubernetes-federation-e2e-gce.env
+
+### Results
+
+Aggregated results are available on the Gubernator dashboard page for
+the federation presubmit tests.
+
+https://k8s-gubernator.appspot.com/builds/kubernetes-jenkins/pr-logs/directory/pull-kubernetes-federation-e2e-gce
+
+### Metrics
+
+We track the flakiness metrics of all the presubmit jobs and
+individual tests that run against PRs in
+[kubernetes/kubernetes](https://github.com/kubernetes/kubernetes).
+
+* The metrics that we track are documented in https://github.com/kubernetes/test-infra/blob/0c56d2c9d32307c0a0f8fece85ef6919389e77fd/metrics/README.md#metrics.
+* Job-level metrics are available in - http://storage.googleapis.com/k8s-metrics/job-flakes-latest.json.
+* As of this writing, federation presubmits have a success rate of
+  93.4%
+
+### Playbook
+
+#### Triggering a new deploy job run
+
+Please ping someone who has access to the Jenkins UI/dashboard and ask
+them to login and click the "Build Now" link on the Jenkins page
+corresponding to the CI job you want to manually start.
+
+#### Triggering a new deploy job run
```

Comment:
I almost thought there was nothing that needed fixing. Had to check a few times :D 

Thanks for catching that. Fixed.